delivery/repositories/project_repository.py

```diff
@@ -173,7 +173,10 @@ def _file_object_from_path(file_path):
 
         checksums = checksums or {}
         if self.filesystem_service.exists(self.multiqc_report_path(project)):
+            log.info("MultiQC reports found in Unaligned/{}, overriding organisation of seqreports".format(project.name))
             return list(map(_file_object_from_path, self.multiqc_report_files(project)))
+        if self.filesystem_service.exists(self.seqreports_path(project)):
```

Comment:
Perhaps you could add an info level log message here as well? It could be nice to se.